plugins/tabs_enabler.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 10 July 2023
+
+@author: Canberk Akın
+
+Gives ability to enable the disabled tabs via a button under Help > Development.
+
+Copyright © 2023 Canberk Akın, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU
+General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not,
+see http://www.gnu.org/licenses/.
+"""
+
+from odemis.gui.plugin import Plugin
+
+class TabEnablerPlugin(Plugin):
+    name = "Tab Enabler"
+    __version__ = "1.0"
+    __author__ = u"Canberk Akın"
+    __license__ = "Public domain"
+
+    def __init__(self, microscope, main_app):
+        super(TabEnablerPlugin, self).__init__(microscope, main_app)
```

Comment:
Now, you can just write:
```suggestion
        super().__init__(microscope, main_app)
```